src/groups/bmq/bmqu/bmqu_alignedprinter.t.cpp

```diff
@@ -0,0 +1,166 @@
+    fields.emplace_back("QueueKey");
+    fields.emplace_back("Number of AppIds");
+
+    const int            indent = 4;
+    bmqu::AlignedPrinter printer(stream, &fields, indent);
+
+    // Test printing various types
+    bsl::string uri       = "bmq://bmq.tutorial.workqueue/sample-queue";
+    bsl::string queueKey  = "sample";
+    const int   numAppIds = 1;
+
+    printer << uri << queueKey << numAppIds;
+
+    bsl::string output = stream.str();
+
+    // Verify output contains expected elements
+    BMQTST_ASSERT(output.find("Queue URI") != bsl::string::npos);
+    BMQTST_ASSERT(output.find("QueueKey") != bsl::string::npos);
+    BMQTST_ASSERT(output.find("Number of AppIds") != bsl::string::npos);
+    BMQTST_ASSERT(output.find(uri) != bsl::string::npos);
+    BMQTST_ASSERT(output.find(queueKey) != bsl::string::npos);
+    BMQTST_ASSERT(output.find("1") != bsl::string::npos);
+}
+
+static void test2_alignment()
+// ------------------------------------------------------------------------
+// ALIGNMENT
+//
+// Concerns:
+//   Ensure proper alignment of fields with different lengths.
+//
+// Plan:
+//   1. Create fields with varying lengths
+//   2. Print values and verify alignment
+//
+// Testing:
+//   Field alignment with mixed field name lengths
+// ------------------------------------------------------------------------
+{
+    bmqtst::TestHelper::printTestName("ALIGNMENT");
+
+    bmqu::MemOutStream       stream;
+    bsl::vector<bsl::string> fields;
+    fields.emplace_back("Short");
+    fields.emplace_back("Very Long Field Name");
+    fields.emplace_back("Med");
+
+    bmqu::AlignedPrinter printer(stream, &fields, 2);
+
+    printer << "value1" << "value2" << "value3";
+
+    bsl::string output = stream.str();
+
+    // Verify all field names are present
+    BMQTST_ASSERT(output.find("Short") != bsl::string::npos);
+    BMQTST_ASSERT(output.find("Very Long Field Name") != bsl::string::npos);
+    BMQTST_ASSERT(output.find("Med") != bsl::string::npos);
+
+    // Basic structure check - should contain colons for alignment
+    BMQTST_ASSERT(output.find(": value1") != bsl::string::npos);
+    BMQTST_ASSERT(output.find(": value2") != bsl::string::npos);
+    BMQTST_ASSERT(output.find(": value3") != bsl::string::npos);
+}
+
+static void test3_emptyFields()
+// ------------------------------------------------------------------------
+// EMPTY FIELDS
+//
+// Concerns:
+//   Behavior when fields vector is empty (should assert in debug mode)
+//
+// Plan:
+//   Verify the assertion behavior for empty fields
+//
+// Testing:
+//   Precondition validation
+// ------------------------------------------------------------------------
+{
+    bmqtst::TestHelper::printTestName("EMPTY FIELDS");
+
+    bmqu::MemOutStream       stream;
+    bsl::vector<bsl::string> fields;  // Empty vector
+
+#ifdef BDE_BUILD_TARGET_SAFE_2
+    // In safe mode, this should assert
+    BMQTST_ASSERT_SAFE_FAIL(bmqu::AlignedPrinter(stream, &fields, 4));
+#endif
```

Comment:
I think that this test is not necessary. It checks a basic precondition in the code, and this precondition is just 1 line and much simpler than the test itself.